index.bs

```diff
@@ -1106,7 +1132,7 @@ dictionary XRInputSourceEventInit : EventInit {
 </pre>
 
 <dfn attribute for="XRInputSourceEvent">frame</dfn>
-An {{XRFrame}} that corresponds with the time that the event took place. This frame's {{XRFrame/views}} array MUST be empty.
+An {{XRFrame}} that corresponds with the time that the event took place. Any {{XRViewerPose}} queried from this frame MUST have an empty {{XRViewerPose/views}} array.
```

Comment:
Did I miss where we are specifying when an XRFrame's active boolean is set to true before invoking the input event callback?

Reply:
Added a section to describe that, thanks!